crates/store/re_types/definitions/rerun/blueprint/components/zoom_level.fbs

```diff
@@ -0,0 +1,13 @@
+namespace rerun.blueprint.components;
+
+// ---
+/// A zoom level determines how much of the world is visible on a map.
+table ZoomLevel (
+    "attr.python.aliases": "float",
+    "attr.python.array_aliases": "npt.ArrayLike",
+    "attr.rerun.scope": "blueprint",
+    "attr.rust.derive": "Default"
+) {
+    /// Zoom level: 0 being the lowest zoom level (fully zoomed out) and 22 being the highest (fully zoomed in).
```

Comment:
where do these numbers come from? Are they just implementation details of the walkers library?
If so, I think it would make sense for us to abstract this away and instead use a more normalized range like 0-1. 0 to 22 seems rather arbitrary.

Reply:
tile servers usually implement 18-23 zoom levels starting from zero. 

Google, Mapbox and Azure have max 22 zoom level (except for a few areas where they have 23)

OpenStreetMap maps also vary, but most support levels only up to 18.

with custom maps you go higher. but on zoom level 24 one pixel is 4.77 mm on the map 

in short, there is no theoretical maximum, but drawing a line at 22 is a reasonable choice

Reply:
Yeah this is a tricky topic. Ideally, the max zoom value would be provided by map provider, and then used by the `ZoomLevel` edit ui. @Wumpf can you devise a mechanism that would allow that?

In any case, we're going to have to take shortcuts here. Cut-off at 19 was a bad idea of mine, it's not enough. I will re-instate the cutoff at 22.0 and turn the warning into a `DEBUG` log or something.

This area will need work in the future, including: https://github.com/podusowski/walkers/issues/198

Reply:
alright! wasn't aware those zoom levels are "a thing", makes sense then

Reply:
**EDIT**: just realised that currently walkers has a hard-coded limit at 19.0 for zoom level. We'll need to improve on that, but for now I'm gonna hard code a limit at 19.0 as well and this will silence most of the log noise (which is in `DEBUG` now anyways).